docs/modules/operation/nav.adoc

```diff
@@ -220,14 +220,7 @@
 
 ** xref:operation/timeseries/introduction.adoc[]
 *** xref:operation/timeseries/configuration.adoc[]
-** xref:operation/daemon-config-files/introduction.adoc[]
-** xref:operation/daemon-config-files/eventd.adoc[]
-** xref:operation/daemon-config-files/notifd.adoc[]
-** xref:operation/daemon-config-files/pollerd.adoc[]
-** xref:operation/daemon-config-files/snmppoller.adoc[]
-** xref:operation/daemon-config-files/syslogd.adoc[]
-** xref:operation/daemon-config-files/trapd.adoc[]
-
+** xref:operation/daemon-reload/introduction.adoc[]
```

Comment:
This page probably fits better under the Administration section (and would align with https://issues.opennms.org/browse/NMS-13510)